refarch-backend/pom.xml

```diff
@@ -134,14 +134,14 @@
         </dependency>
 
         <!-- Database -->
-        <dependency>
-            <groupId>com.h2database</groupId>
-            <artifactId>h2</artifactId>
-        </dependency>
         <dependency>
             <groupId>com.mysql</groupId>
             <artifactId>mysql-connector-j</artifactId>
         </dependency>
```

Comment:
Is this JDBC driver also used for Postgres?

I checked the Postgres homepage and they suggest using only `org.postgresql.postgreql` which you already added. I guess we can remove this.

Reply:
Good point removed it.